src/components/Audio.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
+import mute from '../icons/mute.svg';
+import unmute from '../icons/unmute.svg';
+
+function Audio() {
```

Comment:
Also a nit: I would change this component to be called `AudioIcon` or something for consistency with YearbookIcon, and also to give more context on what it is! (in case we make an Audio component later to actually store the audio)